test/integration/http2_integration_test.cc

```diff
@@ -63,6 +63,23 @@ TEST_P(Http2IntegrationTest, Retry) { testRetry(); }
 
 TEST_P(Http2IntegrationTest, RetryAttemptCount) { testRetryAttemptCountHeader(); }
 
+TEST_P(Http2IntegrationTest, DecodeHeadersReturnsStopAll) { testDecodeHeadersReturnsStopAll(); }
```

Comment:
You could define these in once in protocol_integration_test.cc as DownstreamProtocolntegrationTest to get H1/H2 downstream and avoid the cut and paste.

Reply:
Good idea! Fixed.